plugins/scaffolder-react/src/next/hooks/useFormDataFromQuery.ts

```diff
@@ -13,31 +13,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import { JsonValue } from '@backstage/types';
 import qs from 'qs';
-import { useState } from 'react';
 
 /**
  * This hook is used to get the formData from the query string.
  * @alpha
  */
 export const useFormDataFromQuery = (
   initialState?: Record<string, JsonValue>,
-) => {
```

Comment:
What's the reasons for changes here, can we keep the signature the same so it's not breaking but maybe just use `const [initialState] = useFormDataFromQuery` in the `Stepper` instead?

Reply:
Yes that could also be done. I looked in the codebase for other use of `useFormDataFromQuery` and could find none, I also think it's not widely used in consuming projects. Then again, if we can prevent introducing breaking changes, I'm all for it!